documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,304 @@
+[id="rn-26_{context}"]
 = {project-full} 2.6
 
+You can use {the-lc} {project-first} to migrate virtual machines from the following source providers to {virt} destination providers:
+
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
+* Open Virtual Appliances (OVAs) that were created by VMware vSphere
+* Remote {virt} clusters
+
+The release notes describe technical changes, new features and enhancements, known issues, and resolved issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Redesigned the migration plan creation wizard and dialog
+
+The user interface console has improved the process of creating a migration plan.
+The new migration plan dialog enables faster creation of migration plans. 
+
+It includes only the minimal settings that are required, while you can confirgure advanced settings separately. The new dialog also provides defaults for network and storage mappings, where applicable. The new dialog can also be invoked from the the `Provider` > `Virtual Machines` tab, after selecting the virtual machines to migrate. It also better aligns with the user experience in the OCP console.
+
+.`virtual machine preferences` have replaced {ocp-name} templates
+
+The `virtual machine preferences` have replaced {ocp-name} templates. {project-short} currently falls back to using {ocp-name} templates when a relevant preference is not available.
+
+Custom mappings of guest operating system type to virtual machine preference can be configured using `config` maps. This is in order to use custom virtual machine preferences, or to support more guest operating system types.
+
+.Full support for migration from OVA
+
+{project-short} migration from OVA is now a fully supported feature, and no longer Technical Preview.
+
+.The VM `Running` state is displayed during the migration
+
+{project-short} shows each VM's migration `Running` state during the migration. This single `Running` operation replaces two previous operations, `Create/Post VM` and Run VM`.  link:https://issues.redhat.com/browse/MTV-794[(MTV-794)]
+
+.The `must-gather` logs can now be loaded only by using the CLI
+
+The {product-short} web console can no longer download logs. With this update, you must download every `must-gather` log image by using CLI commands. for more information, see link:https://github.com/openshift/must-gather-operator[Must Gather Operator].
+
+.Migrated VMs can now be reviewed and configured
+
+The {project-short} web console now displays the name and migration status of each virtual machine (VM). The migrated VMs have the `app: <vmname>` label to help further changes in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
+
+
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This part provides features and enhancements introduced in {project-full} 2.6:
+
+.Validation for RHEL 6 guest operating system is no available
```

Comment:
```suggestion
.Validation for RHEL 6 guest operating system is now available
```
maybe we should also phrase it differently if we present this as a feature, e.g., "Warn when migrating VMs with RHEL 6 guest operating system"